glide.yaml

```diff
@@ -266,7 +266,7 @@ import:
 - package: gopkg.in/inf.v0
   version: v0.9.0
 - package: k8s.io/kubernetes
-  version: v1.3.0-alpha.4
+  version: ~1.3.0
```

Comment:
about the `~`: @s-urbaniak tells me we want to pin concrete versions to avoid non-intended updates when running `glide update`.

But why do we update the kubernetes vendoring in this PR?